app/styles/app.sass

```diff
@@ -3589,7 +3581,7 @@ $dropdown-default-theme: $color-table
 .no-issues
   i
     margin-top: 0
-  padding-bottom: 1em
```

Comment:
is this the indented change? `padding-bottom` was assigned to `.no-issues` not to `i`. if it's required ideally should be different class.